pinot-core/src/main/java/org/apache/pinot/core/operator/blocks/IntermediateResultsBlock.java

```diff
@@ -388,7 +391,6 @@ private DataTable getAggregationGroupByResultDataTable()
       throws Exception {
     String[] columnNames = new String[]{"functionName", "GroupByResultMap"};
     ColumnDataType[] columnDataTypes = new ColumnDataType[]{ColumnDataType.STRING, ColumnDataType.OBJECT};
-
```

Comment:
(nit) Don't remove these empty lines for better readability

Reply:
done